blog/_posts/2018-07-13-team-akaringular.md

```diff
@@ -0,0 +1,36 @@
+---
+**title:** New Tech Journey With RGSoC!
+**layout:** post
+**created_at:** Fri Jul 13 2018
+**permalink:** blog/2018-07-13-team-akaringular
+**current:** blog
+**author:** Rachael Kiteme
+**twitter:** rachaelkiteme
+**categories:**  news
+---
+
+
+#:confetti_ball::computer:New Tech Journey With RGSoC :computer::confetti_ball:
+![New Dawn!](/img/blog/2018/akaringular_begin.jpeg)
```

Comment:
Use 'image-class' here to add image credits. You can check other posts in the repo to know how to add image credits. Also, if the image size is small then make sure that the image is in the center.